packages/node_modules/@webex/webex-core/src/lib/services/interceptors/server-error.js

```diff
@@ -24,7 +24,7 @@ export default class ServerErrorInterceptor extends Interceptor {
    * @returns {Object}
    */
   onResponseError(options, reason) {
-    if ((reason instanceof WebexHttpError.InternalServerError) && options.uri) {
+    if ((reason instanceof WebexHttpError.InternalServerError || reason instanceof WebexHttpError.BadGateway || reason instanceof WebexHttpError.reason instanceof WebexHttpError.ServiceUnavailable) && options.uri) {
```

Comment:
I think you are missing a logical OR here.  Otherwise it's `<boolean> instanceof WebexHttpError.ServiceUnavailable` which will always return `false`.